video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import cv2
+import os
+
+dirname = os.path.dirname(__file__)
+CONFIDENCE_MINIMUM = 0.7
+
+opencv_prototxt = os.path.join(
+        dirname, "deploy.prototxt.txt")
+
+opencv_model = os.path.join(
+        dirname, "res10_300x300_ssd_iter_140000_fp16.caffemodel")
+
+# Load model from disk
+net = cv2.dnn.readNetFromCaffe(opencv_prototxt, opencv_model)
```

Comment:
I'd probably make all the above variables to constants. Meaning that you simply give them all uppercase names.